base/shared/src/main/scala/scalaz/data/Void.scala

```diff
@@ -0,0 +1,13 @@
+package scalaz
+package data
+
+trait VoidFunctions {
+  def unsafeVoid: Void = throw new Void.Absurd
```

Comment:
is that api necessary?

Also I'd have expected `absurd::Void->A` here

Reply:
It might be useful to assert void in `(Int === String) => Void` as an example. Due to Scala's subtyping rules absurd is the same as identity. `absurd` would likely result in DCE warnings.

Reply:
ah yes, `Nothing` was there but I missed it! ;)